### PR TITLE
Civ 5074 add new generic services to dq

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://[tools.hmcts.net](https://tools.hmcts.net/jira/browse/CIV-5074)/jira/browse/CIV-5074

### Change description ###

use a generic service instead of individual.

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Review by laperla11 — APPROVED** [review]

File: src/test/utils/mocks/civilClaimResponseMock.json [hunk 1]
```diff
@@ -39,17 +39,20 @@
       "partyTypeDisplayValue": "Organisation"
     },
     "directionQuestionnaire": {
+      "hearing": {},
+      "vulnerabilityQuestions": {},
       "experts": {
         "expertReportDetails": {
-          "hasExpertReports": "yes",
+          "option": "yes",
```
Comment (GorilaNaranja) [inline src/test/utils/mocks/civilClaimResponseMock.json:46]: YesNo type

File: src/test/unit/views/directionsQuestionnaire/vulnerability.test.ts [hunk 2]
```diff
@@ -119,7 +119,7 @@ describe('Vulnerability View', () => {
           .post('/o/token')
           .reply(200, {id_token: citizenRoleToken});
         app.locals.draftStoreClient = mockCivilClaim;
-        const response = await request(app).post(VULNERABILITY_URL).send({isVulnerable: 'yes'});
+        const response = await request(app).post(VULNERABILITY_URL).send({option: 'yes'});
```
Comment (GorilaNaranja) [inline src/test/unit/views/directionsQuestionnaire/vulnerability.test.ts:122]: YesNo type

File: src/test/unit/views/directionsQuestionnaire/expert-report-details.test.ts [hunk 3]
```diff
@@ -117,7 +117,7 @@ describe('Expert Report Details View', () => {
     beforeEach(async () => {
       const response = await request(app).post(DQ_EXPERT_REPORT_DETAILS_URL)
         .send({
-          hasExpertReports: 'yes',
+          option: 'yes',
```
Comment (GorilaNaranja) [inline src/test/unit/views/directionsQuestionnaire/expert-report-details.test.ts:120]: YesNo type

File: src/test/unit/views/directionsQuestionnaire/determination-without-hearing.test.ts [hunk 4]
```diff
@@ -117,7 +117,7 @@ describe('Determination Without Hearing View', () => {
           .reply(200, {id_token: citizenRoleToken});
         app.locals.draftStoreClient = mockCivilClaim;
         const response = await request(app).post(DETERMINATION_WITHOUT_HEARING_URL)
-          .send({isDeterminationWithoutHearing: 'no'});
+          .send({option: 'no'});
```
Comment (GorilaNaranja) [inline src/test/unit/views/directionsQuestionnaire/determination-without-hearing.test.ts:120]: YesNo type

**Review by GorilaNaranja — APPROVED** [review]

**Pull request merged**